source/README.md

```diff
@@ -0,0 +1,77 @@
+# Core Specification Support
+
+- [MaterialXCore](MaterialXCore): Support for the core MaterialX elements
+and graph traversal.
+- [MaterialXFormat](MaterialXFormat): XML serialization support.
+- [PyMaterialX](PyMaterialX) : Core library Python API support
+
+# Shader Generation Support
+
+## Supported APIs
+-   C++
+-   Python
+
+## Supported Backends
+
+-   Core shader generation API: [MaterialXGenShader](MaterialXGenShader) module
+    -  Refer to document in module folder.
+-   Shader Generation languages supported:
+    -   GLSL
+        -   Version 4 or higher
+        -   Core support in [MaterialXGenGLSL](MaterialXGenGLSL) module
+        -   Autodesk OGSFX Effect support in [MaterialXGenOgsFx](MaterialXGenOgsFx) module
+    -   OSL:
+        -   [MaterialXGenOsl](MaterialXGenOsl) module.
+        -   Uses an implementation which differs from the reference OSL
+            implementation
+
+## Definition Libraries
```

Comment:
Should "Definition Libraries", "Library Tree Structure" and "Unsupported definitions for Shader Generation" be part of this file, or part of documents/Libraries/README.md instead?